README.adoc

```diff
@@ -131,19 +131,7 @@ Building upstream docs
 
 If you're required to build the upstream Administration and User Guides, you can build and view each individual guide upstream: https://docs.openstack.org/doc-contrib-guide/docs-builds.html 
 
-If you want to build the SUSE version, equivalent to what is published at at suse.com/documentation, perform the following:
-
-1. Clone https://github.com/SUSE-Cloud/doc-cloud-upstream
-2. Change into the `/scripts/` directory:
-   
-   $ cd scripts
-   
-3. Run `./gen_docs.sh`
-
-   This process clones every repository required, and builds them using the `suse_sphinx_theme`. You need to allocate at
-   least an hour for the process to complete.
-   
-4. You can find the finished product in the `/scripts/build/` folder.
+If you want to build the SUSE version, equivalent to what is published at at suse.com/documentation, see the instructions on how to build here: https://github.com/SUSE-Cloud/doc-cloud-upstream/README
```

Comment:
This link does not work. :(
There is a README file in the `rocky` branch but not in `master`.
(Also, earlier, I failed to grasp that the README was in a different branch and not in a different directory :woman_facepalming: .)

Reply:
Ah yes thank you, good catch. The branch needs to stay as is. It's the main branch of the repo, not master.